pkg/kubectl/resource_printer.go

```diff
@@ -2078,8 +2078,16 @@ func printNetworkPolicyList(list *extensions.NetworkPolicyList, w io.Writer, opt
 	return nil
 }
 
+func hasDefaultAnnotation(obj api.ObjectMeta) bool {
```

Comment:
This is basically a duplicate of the `isDefaultAnnotation()` function above. So it should disappear when you move the generic function which returns a `bool`